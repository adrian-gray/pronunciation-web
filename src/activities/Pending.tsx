import React from "react";

import { capitalise } from "./../utils/utils";

export default (props: { name: string }) => {
  const { name } = props;

  return (
    <div className="head-space">
      <div>
        <h3 className="personals-pace">
          {`${capitalise(name)} - activity coming soon`}
        </h3>
      </div>
    </div>
  );
};
