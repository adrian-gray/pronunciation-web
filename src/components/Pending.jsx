import React from 'react'

import {
  Typography,
  withStyles
} from '@material-ui/core'

const styles = (theme) => ({
  headspace: theme.headspace,
  largeText: theme.largeText
})

const Pending = (props) => {
  const { classes } = props

  return (
    <div className={classes.headspace}>
      <Typography variant='title' gutterBottom>
        {'Pending Activity'}
      </Typography>
    </div>
  )
}

export default withStyles(styles)(Pending)
