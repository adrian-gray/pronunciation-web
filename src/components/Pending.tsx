import React from "react";

import Container from "react-bootstrap/Container";

import { capitalise } from "./../utils/utils";

export default props => {
  const { name } = props;

  return (
    <div className="head-space">
      <Container>
        <h3 className="personals-pace">
          {`${capitalise(name)} - activity coming soon`}
        </h3>
      </Container>
    </div>
  );
};
