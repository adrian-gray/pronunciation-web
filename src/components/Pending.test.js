import React from "react";
import { shallow } from "enzyme";
import { render, cleanup } from "@testing-library/react";

import Pending from "./Pending";

afterEach(cleanup);

test("Pending", async () => {
  const props = {};

  const container = shallow(<Pending {...props} />);

  expect(container.firstChild).toMatchSnapshot();
});
