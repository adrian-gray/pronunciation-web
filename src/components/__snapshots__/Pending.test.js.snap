// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pending 1`] = `
<div>
  <div
    class="head-space"
  >
    <div>
      <h3
        class="personals-pace"
      >
        Bob's Burgers - activity coming soon
      </h3>
    </div>
  </div>
</div>
`;
